public/static/content/fr/terms-and-conditions.md

```diff
@@ -118,4 +116,4 @@ Bon nombre de plateformes de médias sociaux offrent plusieurs choix de langues
 
 
 
-**Dernière mise à jour :** 14 décembre 2023
\ No newline at end of file
+**Dernière mise à jour :** 24 Janvier 2024
```

Comment:
Why is en 

**Last update:** April 24, 2024

And this Jan?

Reply:
Hmmmm... (insert tumble weeds)